api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -277,4 +283,30 @@ public boolean isUserMemberOfGroup(String groupName) {
       return false;
     });
   }
+
+  @Override
+  public NihStatus getNihStatus() {
+    NihApi nihApi = nihApiProvider.get();
+    return retryHandler.run((context) -> {
+      try {
+        return nihApi.nihStatus();
+      } catch (ApiException e) {
+        if (e.getCode() == HttpStatusCodes.STATUS_CODE_NOT_FOUND) {
+          return null;
+        } else {
+          throw e;
+        }
+      }
+    });
+  }
+
+  @Override
+  public void postNihCallback(JWTWrapper wrapper) {
+    NihApi nihApi = nihApiProvider.get();
+    retryHandler.run((context) -> {
+      nihApi.nihCallback(wrapper);
+      return null;
```

Comment:
Since the return type is void, just return should be fine.

Reply:
Unfortunately, that doesn't compile. It has to return something from inside the retry handler. You can see this same pattern in other void methods in this class.